test/server/cards/03_TWI/units/EncouragingLeadership.spec.ts

```diff
@@ -0,0 +1,56 @@
+describe('Encouraging Leadership', function () {
+    integration(function (contextRef) {
+        it('should give +1/+1 for this phase to allies when played', function () {
+            contextRef.setupTest({
+                phase: 'action',
+                player1: {
+                    hand: ['wampa', 'encouraging-leadership'],
+                    spaceArena: ['green-squadron-awing', 'imperial-interceptor'],
+                    groundArena: ['atst']
+                },
+                player2: {
+                    spaceArena: ['star-wing-scout'],
+                    groundArena: ['specforce-soldier'],
+                },
+            });
+            const { context } = contextRef;
+
+            context.player1.clickCard(context.encouragingLeadership);
+
+            // Check if effect is applied to allies
+            expect(context.atst.getPower()).toEqual(7);
+            expect(context.atst.getHp()).toEqual(8);
+            expect(context.greenSquadronAwing.getPower()).toEqual(2);
+            expect(context.greenSquadronAwing.getHp()).toEqual(4);
+            expect(context.imperialInterceptor.getPower()).toEqual(4);
+            expect(context.imperialInterceptor.getHp()).toEqual(3);
+
+            // Check if effect is not applied to non-allies
+            expect(context.starWingScout.getPower()).toEqual(4);
+            expect(context.starWingScout.getHp()).toEqual(1);
+            expect(context.specforceSoldier.getPower()).toEqual(2);
+            expect(context.specforceSoldier.getHp()).toEqual(2);
+
+
+            context.player2.passAction();
```

Comment:
Two small requests then we should be done, thanks for taking the care to sort all this out:
1. Can you remove one of the newlines above so there's not two in a row
2. For the benefit of other people who might use this test as a reference, can you add a comment here that says "test that a unit with the buff can be defeated without causing any errors"

Reply:
All my pleasure :)